packages/perseus/src/util/interactive.ts

```diff
@@ -196,14 +196,11 @@ _.extend(GraphUtils.Graphie.prototype, {
      * Get mouse coordinates in pixels
      */
     getMousePx: function (event: Readonly<{pageX?: number; pageY?: number}>) {
-        const graphie = this;
+        // @ts-expect-error - TS2532 - Object is possibly 'undefined'.
+        const mouseX = event.pageX - $(this.el).offset().left;
 
-        const mouseX =
-            // @ts-expect-error - TS2532 - Object is possibly 'undefined'.
-            event.pageX - $(graphie.raphael.canvas.parentNode).offset().left;
-        const mouseY =
-            // @ts-expect-error - TS2532 - Object is possibly 'undefined'.
-            event.pageY - $(graphie.raphael.canvas.parentNode).offset().top;
+        // @ts-expect-error - TS2532 - Object is possibly 'undefined'.
```

Comment:
I suspect these TS errors have changed. Are they still `Object is possibly 'undefined'`?

Reply:
Yes.  There's two errors that were being hidden. I've broken out the lines so that it's possible to see the two suppressions.  I'm not planing to resolve them though as this is the way the original code is (I just changed it from reaching into Raphael `graphie.raphael.canvas.parentNode` to `this.el`).